django/db/migrations/graph.py

```diff
@@ -138,6 +139,7 @@ def remove_replaced_nodes(self, replacement, replaced):
         for replaced_key in replaced:
             self.nodes.pop(replaced_key, None)
             replaced_node = self.node_map.pop(replaced_key, None)
+            self.replaced_node_keys.add(replaced_key)
```

Comment:
nitpick: This line woud be slightly more logical placed above the preceding one, grouping the manipulation of `replaced_key`, then `replaced_node`.